fauna/client/client.py

```diff
@@ -179,6 +179,34 @@ def get_query_timeout(self) -> Optional[timedelta]:
         else:
             return None
 
+    def paginate(
+        self,
+        fql: Query,
+        opts: Optional[QueryOptions] = None,
+    ) -> "QueryIterator":
+        """
+        Run a query on Fauna and returning an iterator of results. If the query
+        returns a Page, the iterator will fetch additional Pages, until 
+
+        :param fql: A string, but will eventually be a query expression.
```

Comment:
```suggestion
        :param fql: A Query
```

and let's fix this desc in the query() method as well